packages/react/src/experimental/F0VersionHistory/VersionItem/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { F0AvatarPerson } from "@/components/avatars/F0AvatarPerson"
+import { OneEllipsis } from "@/components/OneEllipsis/OneEllipsis"
+import { cn } from "@/lib/utils"
+
+interface VersionItemProps {
+  author: { firstName: string; lastName: string; src?: string }
+  date: string
```

Comment:
Should the date be consistent in all uses of this component? Also, it being a string means that teams could put whatever they want here, not just a date. Same for the time.